fipy/viewers/matplotlibViewer/matplotlib2DContourViewer.py

```diff
@@ -18,10 +18,8 @@ class Matplotlib2DContourViewer(AbstractMatplotlib2DViewer):
     .. _Matplotlib: http://matplotlib.sourceforge.net/
     """
 
-    __doc__ += AbstractMatplotlib2DViewer._test2D(viewer="Matplotlib2DContourViewer")
-
-
-    def __init__(self, vars, title=None, limits={}, cmap=None, colorbar='vertical', axes=None, number=10, levels=None, figaspect='auto', **kwlimits):
+    def __init__(self, vars, title=None, limits={}, cmap=None, colorbar='vertical',
+                 axes=None, number=None, levels=None, figaspect='auto', **kwlimits):
```

Comment:
Line 33 seems to have a stray "float" in front of it (not a change you made, but could be fixed here). Appears in other files as well.

```
limits : dict, optional
    a (deprecated) alternative to limit keyword arguments
float xmin, xmax, ymin, ymax, datamin, datamax : float, optional
    displayed range of data. Any limit set to
    a (default) value of `None` will autoscale.
```

Reply:
Thanks for that. I've removed everywhere I found it.